config/yurtctl-servant/setup_edgenode

```diff
@@ -119,11 +120,18 @@ setup_yurthub() {
         log "setting up yurthub on nodes of minikube"
         yurthub_yaml=$(echo "$YURTHUB_TEMPLATE" | 
             sed 's|__pki_path__|/var/lib/minikube/certs|')
+        apiserver_addr=https://$KUBERNETES_SERVICE_HOST:$KUBERNETES_SERVICE_PORT_HTTPS
     else
         log "setting up yurthub on nodes of ack"
         yurthub_yaml=$(echo "$YURTHUB_TEMPLATE" | 
             sed 's|__pki_path__|/etc/kubernetes/pki|')
-    fi 
+        apiserver_addr=`cat $KUBELET_CONF | grep "server:" | awk  '{print $2}'`
+    fi
+
+    log "setting up yurthub apiserver addr."
+    yurthub_yaml=$(echo "$yurthub_yaml" |
+    sed 's|__kubernetes_service_addr__|$apiserver_addr|')
+
     echo "$yurthub_yaml" > ${STATIC_POD_PATH}/yurt-hub.yaml
```

Comment:
how about change imagePullPolicy of yurthub container to IfNotPresent?

Reply:
Adjusted